cubeviz/layout.py

```diff
@@ -276,6 +277,10 @@ def _toggle_toolbars(self):
         for viewer in self.cube_views:
             viewer._widget.toolbar.setVisible(self._toolbars_visible)
 
+    def _toggle_hover_value(self):
+        for viewer in self.cube_views:
+            viewer._widget._is_tooltip_on = not viewer._widget._is_tooltip_on
```

Comment:
Just a nit to pick, but it seems like the layout should not be updating a private attribute of the `ImageViewer` object. Instead, maybe add a method to `ImageViewer` for toggling this value? Or make it an `@property` of `ImageViewer`?

Reply:
@drdavella: The thing is we don't want anyone to have access to that attribute. Because, if they programmatically change it, there will be a miss match between the actual state of the viewer and the check box in the view menu. I made it private to hide the attribute in the terminal and as a warning.